MAINTAINER.md

```diff
@@ -0,0 +1,334 @@
+      go out in the next release or not too long afterwards.
+
+    - [`important-longterm`](https://github.com/weaveworks/eksctl/labels/priority%2Fimportant-longterm):
+      Examples include larger features which need time to design, things which contribute
+      to the goals/wellbeing of the project but are not massively urgent, infrequent test flakes
+      and other useful things which we really want but can live without for the moment.
+      These things should be implemented and released within a couple of months
+      of the request.
+
+    - [`backlog`](https://github.com/weaveworks/eksctl/labels/priority%2Fbacklog).
+      Things which would be nice to have but will survive without.
+
+1. Consider the "bar of entry". Is the task really simple, with tons of detail, minimal scope
+  and maybe even some hints on how to implement? Assign a `good-first-issue` label.
+  Does it have potential to be a `good-first-issue`, but a little more context is needed?
+  Either provide that yourself or ask the OP to elaborate.
+
+    These issues should be accessible to folks new to the repo, as well as to open source in general.
+    They should present a low/non-existent barrier to entry with a thorough description,
+    easy-to-follow reproduction (if relevant) and enough context for anyone to pick up.
+    The objective should be clear, possibly with a suggested solution or some pseudocode.
+    If anything similar has been done, that work should be linked.
+
+    If the bar is a little higher, add a `help wanted` label.
+
+1. Add the issue to the `eksctl-backlog` if it looks like a good candidate
+  for the upcoming sprint.
+
+1. **Respond to the issue**! One of the biggest reasons that people lose interest
+  in open-source projects is lack of interaction. It also does not help the project's
+  reputation when people feel ignored or do not feel welcome.
+  Even if you have absolutely nothing useful to say, there should always
+  be _at least_ one comment on every issue (that was not written by the OP).
+
+    Here is some boilerplate if you are lost for words:
+
+    - "Hi @\<op-name\>! Thanks for reporting this bug :+1: If you have a fix in mind,
+      we are happy to accept PRs, otherwise the core team will get to it soon."
+
+    - "Hi @\<op-name\>! Thanks for opening this feature request :+1: The team will review
+      and see whether this functionality fits in with the project. In the meantime,
+      if you would like to demonstrate a solution, please open a PR :smiley: ."
+
+    - "Hi @\<op-name\>! Thanks for creating a help ticket. I will do some research and get
+      back to you asap!"
+
+
+**Notes:**
+
+Try not to forget about issues. If you don't know what they are asking for,
+don't understand the bug, or don't even know if it is a thing we should care about,
+tag in the rest of the team on slack after you have written your boilerplate "Thanks for opening"
+message.
+
+
+## Plan for a sprint
```

Comment:
I'm not sure this needs to be in the MAINTAINER doc necessarily? Even if we had external maintainers, I don't know if they'd be involved in everything here.

Reply:
Yeh i was kindof going back and forth on this. Stuff in the MAINTAINER.md is more of a handbook for just us, and i used the work Maintainer as it is part of the OS language. We could also move that entire file into a `eksctl-handbook` repo which is private and just for us?

Reply:
👍  I think private makes sense here

Reply:
cool. @cPu1 @aclevername do you both agree that having a private `eksctl-handbook` repo and moving the MAINTAINER.md and contents of the handbook googledoc is a better solution here?

We would have to be very strict on what we keep in there, err on the side of public where possible, but I think mike is right: until we have external maintainers we can afford to have it private. However, it does depend on what part AWS intends to play, tho i suppose we can give them access to that private repo if they are going to be maintaining with us. WDYT?

Reply:
Private repo sounds good 👍

Reply:
note to self: also add 'how to' on bumping eksctl-build image

Reply:
To be clear, I think some of this is really good to have public, for example the label semantics and the release process. I'm not sure where the line is though..

Reply:
To be clear, I think some of this is really good to have public, for example the label semantics and the release process. I'm not sure where the line is though.. (deleted and reposted same message from earlier)

Reply:
https://github.com/weaveworks/corp/issues/1362

Reply:
Okay, for now I am gonna put everything in Maintainers into that new repo and decide later what to move publicly again. I want to get the other stuff merged this week if poss 👍  make sense?